CDS/js/HMACRequest.js

```diff
@@ -0,0 +1,28 @@
+var reqwest = require('reqwest');
```

Comment:
I don't think there is any other reason not to than just being consistent with the js already here.